calc-fee/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "calc-fee"
+version = "0.1.0"
+authors = ["Alexander Theißen <alexander.theissen@parity.io>"]
+edition = "2018"
+license = "Apache-2.0"
+publish = false
+repository = "https://github.com/polkadot-js/wasm/"
+description = "Calculate extrinsic fees off-chain"
+
+[lib]
+crate-type = ["cdylib"]
+
+[features]
+debug = ["console_error_panic_hook"]
+
+[dependencies]
+wasm-bindgen = { version = "0.2", default_features = false, features = ["serde-serialize"] }
+serde_derive = { version = "1", default_features = false }
+serde = { version = "1", default_features = false }
+console_error_panic_hook = { version = "0.1", optional = true }
+sp-arithmetic = { git = "https://github.com/paritytech/substrate", default_features = false }
```

Comment:
We should replace that by a released version as soon as that is possible. The current release on crates.io (2.0.0-rc2) has a broken Fixed128 impl that does not work with negative coefficients on unsigned Balances.